arches/app/media/js/views/components/etl_modules/bulk-data-deletion.js

```diff
@@ -144,21 +144,25 @@ define([
             }
         });
 
+        this.deleteAlert = function() {
+            self.alert(
+                new AlertViewModel(
+                    'ep-alert-red',
```

Comment:
Let's change this to "ep-alert-blue" as it's informational.

Reply:
Changed in latest commit d7537f4327a41714fdb86bb64a9b79ee43543fea